src/Features/CSharp/Portable/CodeRefactorings/SyncNamespace/CSharpChangeNamespaceService.cs

```diff
@@ -470,16 +443,25 @@ private static ExpressionSyntax CreateNamespaceAsMemberAccess(ImmutableArray<str
         /// trivia around closing brace are concatenated together respectively.
         /// </summary>
         private static (ImmutableArray<SyntaxTrivia> openingTrivia, ImmutableArray<SyntaxTrivia> closingTrivia)
-            GetOpeningAndClosingTriviaOfNamespaceDeclaration(NamespaceDeclarationSyntax namespaceDeclaration)
+            GetOpeningAndClosingTriviaOfNamespaceDeclaration(BaseNamespaceDeclarationSyntax baseNamespace)
         {
             var openingBuilder = ArrayBuilder<SyntaxTrivia>.GetInstance();
-            openingBuilder.AddRange(namespaceDeclaration.GetLeadingTrivia());
-            openingBuilder.AddRange(namespaceDeclaration.OpenBraceToken.LeadingTrivia);
-            openingBuilder.AddRange(namespaceDeclaration.OpenBraceToken.TrailingTrivia);
-
             var closingBuilder = ArrayBuilder<SyntaxTrivia>.GetInstance();
```

Comment:
nit: This is unused for the FileScopedNamespace code path. Consider constructing a builder only for the file-scoped case.